ios/keyman/Keyman/Keyman/KMWebBrowser/WebBrowserViewController.swift

```diff
@@ -204,6 +204,36 @@ class WebBrowserViewController: UIViewController, UIWebViewDelegate, UIAlertView
   func webView(_ webView: UIWebView,
                shouldStartLoadWith request: URLRequest,
                navigationType: UIWebView.NavigationType) -> Bool {
+    if request.url?.lastPathComponent.hasSuffix(".kmp") ?? false {
+      // Can't have the browser auto-download with no way to select a different page.
+      // Can't just ignore it, either, as the .kmp may result from a redirect from
+      // the previous URL.  (Like if using the keyman.com keyboard search!)
+      let userData = UserDefaults.standard
+      userData.set(nil as String?, forKey: webBrowserLastURLKey)
+      userData.synchronize()
+
+      // The user is trying to download a .kmp, but the standard
+      // UIWebView can't handle it properly.
+
+      ResourceDownloadManager.shared.downloadRawKMP(from: request.url!) { file, error in
+        // do something!
+        if let error = error {
+          let alert = ResourceFileManager.shared.buildSimpleAlert(title: "Error", message: error.localizedDescription)
```

Comment:
Yes, that `"Error"` should be localized.  Sadly, I noted that there are a couple of other matching, non-localized spots yet to handle... which are actually rather related:

https://github.com/keymanapp/keyman/blob/0801d69bbc80c6129974688d347e31fc851042d4/ios/engine/KMEI/KeymanEngine/Classes/Resource%20Management/ResourceFileManager.swift#L267-L282

(Lines 268 and 274, in particular.)

May as well address all three together, whether that be within this PR or a separate one.